graviti/dataframe/frame.py

```diff
@@ -137,6 +137,9 @@ def __getitem__(self, key: Union[str, Iterable[str]]) -> Container:
         raise KeyError(key)
 
     def __setitem__(self, key: str, value: Union[Iterable[Any], _C]) -> None:
+        if self.operations is not None and key in self._column_names:
+            raise NotImplementedError("Replacing a column is not supported yet")
```

Comment:
do we need to mention the dataframe is a sheet in commit, not a local dataframe?